client/my-sites/theme/main.jsx

```diff
@@ -125,8 +126,16 @@ const ThemeSheet = React.createClass( {
 		);
 	},
 
+	getFullLengthScreenshot() {
+		/**
+		 * Location of full length theme screenshot in screenshots array
+		 * depends on themes API v1.2 screenshots field
+		 */
+		return this.props.screenshots[ 0 ];
```

Comment:
This array access causes problems for me before the theme data is loaded.

Testcase
* new tab
* go to /themes
* click on any theme